packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/ledger/pathSelection.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback } from 'react'
+import { useRouter } from 'expo-router'
+import { DerivationPathSelector } from 'new/features/ledger/components/DerivationPathSelector'
+import { useLedgerSetupContext } from 'new/features/ledger/contexts/LedgerSetupContext'
+import { LedgerDerivationPathType } from 'services/ledger/types'
+
+export default function PathSelectionScreen(): JSX.Element {
```

Comment:
you should create a `screens` folder inside `new/features/ledger`, move all the screen components there and just reference them here. that's the pattern we have been using.